src/inputs/input-with-icon.scss

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './input';
+
+// Mixin to have inputs combined with icon (eg. select)
+@mixin iui-input-with-icon {
+  @include iui-input;
+  display: grid;
+  align-items: center;
+  grid-template: 'input icon' / 1fr auto;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+  position: relative;
+  width: 100%;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: border-color $iui-speed-fast ease-out;
+  }
+
+  &:hover {
+    @include themed {
+      border-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
+    }
+
+    &::after {
+      border-bottom: none;
+      @include themed {
+        border-top: $iui-after-border t(iui-icons-color-actionable-hover);
+      }
+    }
+  }
+
+  &.iui-active {
+    &::after {
+      border-top: none;
+      @include themed {
+        border-bottom: $iui-after-border t(iui-icons-color-actionable);
+      }
+    }
+
+    &:hover::after {
+      @include themed {
+        border-bottom: $iui-after-border t(iui-icons-color-actionable-hover);
+      }
+    }
+  }
+
+  &.iui-disabled {
+    @include themed {
+      background-color: t(iui-color-background-disabled);
+      border-color: t(iui-color-background-disabled);
+    }
+
+    .iui-end-icon {
+      cursor: not-allowed;
```

Comment:
I think this cursor styling is in the wrong place. It seems that `iui-input-with-icon` is just missing the entire cursor styling, so if you go in the demo and hover over non-labeled selects, then you get the default cursor.

Reply:
The cursor styling is mostly fixed now, but the icon is now showing the `pointer` cursor even when disabled

Reply:
Added cursor styling to iui-input-icon-inline disabled state